app/build.gradle.kts

```diff
@@ -70,6 +70,11 @@ android {
 
             applicationIdSuffix = ".debug"
             resValue("string", "app_name", "Ivy Wallet Debug")
+
+            firebaseCrashlytics {
+                mappingFileUploadEnabled = false
```

Comment:
This removes the deobfuscation mapping. I.e., still uploads crashes but as gibberish.